activesupport/lib/active_support/core_ext/module/remove_method.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/module/redefine_method"
```

Comment:
Compatibility, because that file now contains `redefine_method`, which was previously in here.